File: javascript/7kyu/makeBackronym.js
// https://www.codewars.com/kata/55805ab490c73741b7000064

//     An acronym deliberately formed from a phrase whose initial letters spell out a particular word or words, either to create a memorable name or as a fanciful explanation of a word's origin.

function makeBackronym(string){
    return string.toUpperCase().split('').map(function(s){
      return dict[s];
    }).join(' ');
  }

File: java/7kyu/makeBackronym.java
// https://www.codewars.com/kata/55805ab490c73741b7000064

import java.util.Map;
import java.util.stream.Collectors;

public class Backronym {
  private static Map<String, String> dictionary = Preload.dictionary;
  public static String makeBackronym(String acronym) {
    return acronym.toUpperCase().chars().mapToObj(i -> dictionary.get("" + (char)i)).collect(Collectors.joining(" "));
  }
}